LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Mac/Cid.cs

```diff
@@ -3,17 +3,17 @@
 
 namespace LoRaTools
 {
+#pragma warning disable CA1008 // Enums should have zero value
```

Comment:
Consider adding a justification:

```suggestion
#pragma warning disable CA1008 // Enums should have zero value (but values reflect CID from spec which has none)
```